elasticsearch/src/main/java/org/hibernate/search/elasticsearch/settings/impl/model/AnalyzerDefinitionJsonAdapterFactory.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.elasticsearch.settings.impl.model;
+
+import java.util.List;
+
+import com.google.gson.reflect.TypeToken;
+
+/**
+ * @author Yoann Rodiere
+ */
+public class AnalyzerDefinitionJsonAdapterFactory extends AnalysisDefinitionJsonAdapterFactory {
```

Comment:
Wondering if it's worth introducing this infrastructure rather than simply use a JsonElement/JsonArray to serialize the elements.